queries.sql

```diff
@@ -28,4 +28,49 @@ WHERE name <> 'Gabumon';
 
 SELECT *
 FROM animals
-WHERE weight_kg > 10.4 AND weight_kg < 17.3;
\ No newline at end of file
+WHERE weight_kg > 10.4 AND weight_kg < 17.3;
+
+BEGIN;
+
+UPDATE animals
+  SET species = 'unspecified';
+
+ROLLBACK;
+
+BEGIN;
+
+UPDATE animals
+  SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+UPDATE animals
+  SET species = 'pokemon'
+WHERE species IS NULL;
+
+COMMIT;
+
+BEGIN;
+
+DELETE FROM animals;
+
+ROLLBACK;
+
+BEGIN;
+
+DELETE FROM animals
+WHERE date_of_birth > 'Jan 1, 2022';
+
+SAVEPOINT sp1;
+
+UPDATE animals
+  SET weight_kg = weight_kg * -1;
+
+ROLLBACK TO sp1;
+
+BEGIN;
+
+UPDATE animals
+  SET weight_kg = weight_kg * -1
+WHERE weight_kg < 0;
+
+COMMIT;
```

Comment:
These are transaction queries means these queries are manipulating the data in your database so these queries are supposed to be in data.sql file which is responsible for having data manipulation queries. queries.sql file is just responsible for data retrieval queries (means queries with Select) 👍

Reply:
Please note that in the instructions I clearly say that these transactions should be in the queries.sql file
![Screenshot from 2022-12-06 22-27-33](https://user-images.githubusercontent.com/69245974/206016219-82f9d83e-5e78-488b-92c9-aa24a12327e7.png)

Reply:
Hey @Daron976 My bad. Its good that you have mentioned this 👍 
Thanks for your feedback. You can leave this change and work on the other changes 😊